en-US/Downloads.md

```diff
@@ -1,15 +1,15 @@
 ---
 layout: default
-title: Downloads
-description: Download Windows for IoT, Visual Studio, Software Development Kits or any of the other tools available on this page to get started developing for the Internet of Things today!
-keyword: dashboard, download, windows 10 iot core, tools
+title: IoT Core Dashboard and Raspberry Pi Downloads | Windows IoT 
+description : Download Windows for IoT, Visual Studio, Software Development Kits or any of the other tools available on this page to get started developing for the Internet of Things today! 
+keywords : setup windows 10 raspberry pi, setup windows 10 dragonboard, setup windows 10 minnowboard max, windows 10 developer program, windows 10 iot core, internet of things, iot, download windows 10 iot core 
 permalink: /en-US/Downloads.htm
 lang: en-US
 ---
 
 <section class="section">
   <header class="page-title-header remove-top-margin">
-    <h1 class="page-title">Downloads and Tools</h1>
+    <h1 class="page-title">Windows IoT Core Downloads and Tools</h1>
```

Comment:
You also need to add Anchor texts for the download links on line # 55, 60 and 65.